app/uk/gov/hmrc/vulnerabilities/model/Vulnerability.scala

```diff
@@ -0,0 +1,85 @@
+/*
+ * Copyright 2022 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.vulnerabilities.model
+
+import play.api.libs.functional.syntax.{toFunctionalBuilderOps, unlift}
+import play.api.libs.json._
+import uk.gov.hmrc.mongo.play.json.formats.MongoJavatimeFormats
+
+import java.time.Instant
+
+case class Vulnerability(
+    service: String,
+    serviceVersion: String,
+    vulnerableComponentName: String,
+    vulnerableComponentVersion: String,
+    componentPathInSlug: String,
+    cve: String,
```

Comment:
AMBER: Might be better to call these `id`, `score` and `description` - so as not to tie ourselves to CVE...

Reply:
Updated :)